CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 ### Minor
 ### Patch
 
+- Adds [Danger](http://danger.systems/js/) to pull requests.
```

Comment:
@chrislloyd Could we add `Build: ` in front of it to match the rest of the changelog?